nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/document.js

```diff
@@ -1,105 +1,78 @@
 'use strict';
 
-let teardownCbs = [];
-
-let doc = {
-  'entity-type': 'document',
-  name: 'doc',
-  properties: {
-    'dc:title': 'document'
-  }
-};
-
 module.exports = function() {
 
-  this.Given(/^I have a (.*) document$/, (docType) => {
+  this.Given(/^I have a(.*)document$/, (docType) => {
+    docType = docType || 'File';
+    let doc = fixtures.documents.init(docType);
+    // create the document
+    return fixtures.documents.create(this.doc.path, doc).then((doc) => {
```

Comment:
Perhaps it would be cleaner to keep a `fixtures.document.current`instead of continuously updating a `this.doc`...